src/util/rangelist.h

```diff
@@ -0,0 +1,90 @@
+#include <util/assert.h>
+
+#include <QList>
+#include <QRegularExpression>
+#include <QString>
+#include <algorithm>
+
+namespace mixxx {
+
+const QString kGroupSeparator = QStringLiteral(", ");
+const QString kRangeSeparator = QStringLiteral(" - ");
+// when changing groupSeparator or rangeSeparator, rangeListMatchingRegex must
+// be adjusted as well.
+const QRegularExpression kRangeListMatchingRegex(
+        QStringLiteral("(?:(\\d+)(?:\\s*-\\s*(\\d+))?)[\\s,]*"));
```

Comment:
Let's not pollute the mixxx namespace with these constants. We can move the implementation into a `rangelist.cpp` file and then put these values in an anonymous namespace.

Reply:
not sure how to declare the functions in the header then, but I'll experiment a bit.

Reply:
Declare the functions but leave out the body (`{ [...] }`). Then put the actual implementation into a cpp file.

Reply:
Ah, so just like in C, makes sense.